src/js/app.js

```diff
@@ -1,98 +1,82 @@
+import {
+  showEditField,
+  toggleCheckBox,
+  appendTaskEl,
+  removeTaskEl,
+} from './todo-helper-functions.js';
+import addDraggableListener from './dragEffect.js';
+
 const tasksListEl = document.getElementById('tasksList');
 const clearListBtn = document.getElementById('clear-list');
 const form = document.querySelector('form');
 
-let toDoList = JSON.parse(localStorage.getItem('todo')) || [];
+let toDoList;
 
-toDoList = [
-  {
-    description: 'Do my math homework',
-    isCompleted: false,
-    id: 9877,
-  },
-  {
-    description: 'Take out the trash',
-    isCompleted: false,
-    id: 9878,
-  },
-  {
-    description: 'Feed my cats',
-    isCompleted: false,
-    id: 9999,
-  },
-];
+const handleClearListBtnClick = (displayToDoList) => {
+  clearListBtn.addEventListener('click', () => {
+    const filteredTaskList = toDoList.filter((task) => !task.isCompleted);
+    toDoList = filteredTaskList.reduce((tasks, task, idx) => {
+      const next = idx + 1;
+      return [...tasks, { ...task, index: next }];
+    }, []);
```

Comment:
[Optional] You have done great here at using the reduce function to reset the indexes of the tasks. I think that using the map function would be more appropriate since you're not really performing a reduction. It will also give a shorter code.

Something like this would work:
`toDoList = filteredTaskList.map((task, idx) => ({...task, index: idx+1}));`